PanelSwCustomActions/DismSched.cpp

```diff
@@ -78,6 +78,10 @@ extern "C" UINT __stdcall DismSched(MSIHANDLE hInstall)
 		ExitOnFailure(hr, "Failed to get Cost.");
 		hr = WcaGetRecordInteger(hRecord, 7, &nErrorHandling);
 		ExitOnFailure(hr, "Failed to get ErrorHandling.");
+		hr = WcaGetRecordInteger(hRecord, 8, &nEnableAll);
+		ExitOnFailure(hr, "Failed to get EnableAll.");
+		hr = WcaGetRecordInteger(hRecord, 9, &nOrder);
```

Comment:
Same comment as previous one- is there any use for the Order field other than in the ORDER BY clause?

Reply:
I will remove it

Reply:
removed